src/modules/Routing.rb

```diff
@@ -50,8 +50,9 @@ class RoutingClass < Module
     # "routes" file location
     ROUTES_FILE = "/etc/sysconfig/network/routes"
 
-    SYSCTL_IPV4_PATH = ".etc.sysctl_conf.\"net.ipv4.ip_forward\""
-    SYSCTL_IPV6_PATH = ".etc.sysctl_conf.\"net.ipv6.conf.all.forwarding\""
+    SYSCTL_AGENT_PATH = ".etc.sysctl_conf"
+    SYSCTL_IPV4_PATH = "net.ipv4.ip_forward"
+    SYSCTL_IPV6_PATH = "net.ipv6.conf.all.forwarding"
```

Comment:
I think this is VERY confusing. How should I know which of these to use?
```rb
SCR.Read(path(SYSCTL_IPV6_PATH))
SCR.Read(path("#{SYSCTL_AGENT_PATH}.#{SYSCTL_IPV6_PATH}"))
SCR.Read(path("#{SYSCTL_AGENT_PATH}.\"#{SYSCTL_IPV6_PATH}\""))
```

Thinking of it, the best seems
```rb
SYSCTL_AGENT_PATH = path(".etc.sysctl_conf")
SYSCTL_IPV4_PATH = SYSCTL_AGENT_PATH + "net.ipv4.ip_forward"  # add as a single path component
SYSCTL_IPV6_PATH = SYSCTL_AGENT_PATH + "net.ipv6.conf.all.forwarding"
...
SCR.Read(SYSCTL_IPV6_PATH)
```

(There is a difference between `path(".foo") + ".bar.baz"` and `path(".foo") + path(".bar.baz")` which I will need to document properly, [`Yast.Path#+`](http://www.rubydoc.info/github/yast/yast-ruby-bindings/Yast%2FPath%3A%2B) does not mention it)